etl/socrata_export/socrata_export.py

```diff
@@ -21,12 +22,21 @@
         "typename": "socrata_export_crashes_view",
         "dataset_id": SOCRATA_DATASET_CRASHES,
         "query": QUERIES["crashes"],
+        "batch_size": DEFAULT_RECORD_BATCH_SIZE,
     },
     {
         "name": "people",
         "typename": "socrata_export_people_view",
         "dataset_id": SOCRATA_DATASET_PEOPLE,
         "query": QUERIES["people"],
+        "batch_size": DEFAULT_RECORD_BATCH_SIZE,
+    },
+    {
+        "name": "crash_components",
+        "typename": "moped_component_crashes",
+        "dataset_id": SOCRATA_DATASET_CRASH_COMPONENTS,
+        "query": QUERIES["crash_components"],
+        "batch_size": 10000,
```

Comment:
will this be replacing/upserting all 700k records daily? I don't really understand what the new record ID is doing. Wondering if we can avoid having to do all that replacing

Reply:
Also, this script uses `socrata_client.upsert` which should be unnecessary if we're truncating it every time we run the script. `insert` should be a lot faster, especially with 700k records

Reply:
yeah, that's good point—switching to insert should be no problem. i'm not sure if it will actually have an performance improvement, but that would be quite nice 🤞 

but yeah, we are replacing 700k each time this runs, although we could schedule weekly or monthly. the new record ID enables us to download -> upload in batches by providing us with an ID we can sort and query on. it's basically a hack to make this dataset play with the existing socrata ETL.

i'm not really sure how to avoid replacing this dataset without some pretty complex engineering 🤔  

we could abandon this route and connect directly to the DB?

Reply:
Ah that makes sense. I guess it's fine but I am a little worried about things timing out and not getting full updates, but I could see how much of a headache tracking changes in both moped and VZ would be. I think Socrata makes sense still but if we want to try the dataflow method we can!